src/components/learning-guide-chart.coffee

```diff
@@ -71,38 +66,38 @@ LearningGuide = React.createClass
 
     circles = wrap.enter()
       .append('g')
-      .attr('transform', (f,i)->
+      .attr('transform', (f,i) ->
         "translate(#{points[i].x},#{points[i].y})"
       )
-      .on('click', (field)=>
+      .on('click', (field) =>
         @navigateToPractice(field)
       )
     circles.append('circle')
-      .attr('r', (f)->
+      .attr('r', (f) ->
         # awaiting a better alogorithm for the circle radius
         Math.max(f.questions_answered_count / 20, 2)
       )
     circles.append('text')
-      .text( (f)->
+      .text( (f) ->
         f.questions_answered_count
       )
       .attr('text-anchor', 'middle')
       .attr('dy', '0.5')
 
 
-  drawXAxis: (container, fields, points)->
+  drawXAxis: (container, fields, points) ->
     wrap = container.append('g')
       .attr('class', 'x-axis')
       .selectAll('line')
       .data(fields)
-    me=this
+    me = @ # Since displayUnit needs this and the 'this' scope still needs @parentElement
```

Comment:
Initially I was wondering why you did it for `displayUnit` but not for `navigateToPractice` and then I read a little more ; )